app/scripts/directives/bindService.js

```diff
@@ -207,6 +228,9 @@
         if (ctrl.podPresets) {
           loadApplications();
         }
+        if (ctrl.isMobileEnabled) {
+          loadServiceClasses();
```

Comment:
@maleck13 @sedroche We'll be loading the service classes here instead because the serviceclass for that service instance will be retrieved by this line [here](https://github.com/aerogear/origin-web-console/pull/70/files#diff-9b6377c25f28da45ed50c1d71ef2863aR270) anyway.